src/FWorld.hpp

```diff
@@ -0,0 +1,35 @@
+#ifndef __FWORLD_HPP__
+#define __FWORLD_HPP__
+
+// FCharacter.cpp
+static const bool FACING_LEFT = false;
+static const float RESURRECTION_PERIOD = 3.f;
+static const float MOVEMENT_SPEED = 0.23f;
+static const float JUMPING_PERIOD = 0.4f; 
+static const FVector JUMP_VECTOR(0.f, -0.4f); // needs FVector loaded beforehand
+// AI constants skipped for now
+
+// FEngine.cpp
+static const float CREATING_POWERUP_PROBABILITY = 0.005f;
+static const FVector PROJECTTILE_SIZE(10, 10); // needs FVector loaded beforehand // what about other projectiles?
+static const float PROJECTTILE_TIME = 1000.f; // what about other projectiles?
+static const int PROJECTTILE_DAMAGE = 10; // what about other projectiles?
+// The same for powerup (heal instead of damage) in lines between 115-121, skipped for now
```

Comment:
You see the problem, when we have many projectiles and powerups, this makes little sense. We need a better system than we have, but those constants do not belong in this file. Only `CREATING_POWERUP_PROBABILITY` should stay.